public/index.html

```diff
@@ -6,6 +6,43 @@
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
 		<title>Homepage</title>
 		<link rel="stylesheet" href="../src/css/index.css" />
+		<link
+			rel="stylesheet"
+			href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+		/>
 	</head>
-	<body></body>
+	<body>
+		<header>
+			<form>
+				<div class="navigation">
+					<div class="navigation__search-wrapper">
+						<input
+							type="search"
+							class="navigation__search-input"
+							placeholder="Search..."
+						/>
+						<button class="navigation__search-button" type="submit">
```

Comment:
Just to let you know, quick tip if a button is in a form by default it has type="submit"